modules/client-handler/src/main/java/org/apache/ignite/client/handler/requests/table/ClientTupleGetRequest.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.client.handler.requests.table;
+
+import java.io.IOException;
+import java.util.concurrent.CompletableFuture;
+
+import org.apache.ignite.client.proto.ClientMessagePacker;
+import org.apache.ignite.client.proto.ClientMessageUnpacker;
+import org.apache.ignite.table.manager.IgniteTables;
+
+import static org.apache.ignite.client.handler.requests.table.ClientTableCommon.readTable;
+import static org.apache.ignite.client.handler.requests.table.ClientTableCommon.readTuple;
+import static org.apache.ignite.client.handler.requests.table.ClientTableCommon.writeTuple;
+
+/**
+ * Client tuple get request.
+ */
+public class ClientTupleGetRequest {
+    /**
+     * Processes the request.
+     *
+     * @param in Unpacker.
+     * @param out Packer.
+     * @param tables Ignite tables.
+     * @return Future.
+     * @throws IOException On serialization error.
+     */
+    public static CompletableFuture<Void> process(ClientMessageUnpacker in, ClientMessagePacker out,
+                                                  IgniteTables tables)
+            throws IOException {
```

Comment:
Could you please move IOException to previous line and each input parameter at new line?
According to this rule: https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines#CodingGuidelines-MethodArguments

Reply:
Fixed here and in some other places.